stable.js

```diff
@@ -1099,6 +1099,25 @@ var idrinth = {
         },
         controls: null,
         tooltipTO: null,
+        buildModal:function(title,content,altFunc) {
```

Comment:
You don't handle cases when, for some reasons, content is wrong type. Could you add a type check handle for the function to proceed?